### DISCUSSION TITLE
Make an iOS and iPadOS version of the app

### DISCUSSION THREAD

**Discussion by MysteriousOrb2000** [Docile-Alligator/Infinity-For-Reddit#999] [Not planned Ideas] [answered]
There's no FOSS Reddit app for Apple devices. Infinity for Reddit would be welcomed if it was made for iOS and iPadOS.

**Comment by Wladefant** [comment, marked as answer]
The development of an reddit client for ios would require a completely new framework, new code and a new programming language, and of course a new reddit api wrapper, so it is definitely out of scope
